### PR TITLE
Add a "go fast" keyboard toggle and a fasttape URL parameter.

### LINKED ISSUE #8: Support faster-than-realtime tape loading

### PR DESCRIPTION
**Pull request by scarybeasts**
Fixes #8, #56 

Ctrl + INSERT now toggles "max speed" on or off.
The new URL parameter &fasttape will automatically engage "max speed" when the cassette motor is on.

In both cases, accuracy is unaffected. It's just a case that virtual time runs faster than real time, to the extend of the hardware's capabilities.

On my ancient laptop, "max speed" is about 3x.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -325,6 +330,9 @@ require(['jquery', 'underscore', 'utils', 'video', 'soundchip', 'ddnoise', 'debu
             } else if (code === utils.keyCodes.HOME && evt.ctrlKey) {
                 utils.noteEvent('keyboard', 'press', 'home');
                 stop(true);
+            } else if (code === utils.keyCodes.INSERT && evt.ctrlKey) {
+                utils.noteEvent('keyboard', 'press', 'insert');
```
Comment (mattgodbolt) [inline main.js:334]: Thanks for keeping stats here :) (not that I've looked at them in years...but will do!)

File: main.js [hunk 2]
```diff
@@ -1192,13 +1200,26 @@ require(['jquery', 'underscore', 'utils', 'video', 'soundchip', 'ddnoise', 'debu
                 last = 0;
                 return;
             }
-            requestAnimationFrame(draw);
+            var speedy;
```
Comment (mattgodbolt) [inline main.js]: maybe rephrase as:

```javascript
var speedy = fastAsPossible || (fastTape && processor.acia.motorOn);
if (speedy) { ... }
```

Seeing an uninitialised variable tickles my C++ spider sense (even though it's totally fine in this context :))

Comment (scarybeasts) [reply main.js]: Fixed in latest upload.
